### DISCUSSION TITLE
720p download is no longer working

### DISCUSSION THREAD

**Discussion by Alex-Nothoryus** [FreeTubeApp/FreeTube#5347] [General]
In previous versions, the option to download in 720p was working, which is the minimum acceptable for a video, but now there is only the option to download in 360p, which is a terrible quality, will we have a solution for this soon?

**Comment by absidue** [comment]
Please read my comment here: https://github.com/FreeTubeApp/FreeTube/discussions/5333#discussioncomment-9863599

**Reply by Alex-Nothoryus** [reply]
@absidue So for me it was a useless project since I only used this program to download videos in 720p, but it is a great project, it can still be used to watch videos without advertisements which is already very good, but I think the project has to progress and not regress, and this flaw that no longer allows you to download videos in 720p is a setback for the project

**Reply by absidue** [reply]
Again it's not a flaw in FreeTube that YouTube no longer has the 720p stream with both audio and video in it, the only one that still has both is the 360p one.

**Reply by kommunarr** [reply]
The answer:
> YouTube stopped offering that stream, which is why it is missing. Unfortunately we can't do anything about it to make it come back, sorry.

The response:
> So for me it was a useless project [...] I think the project has to progress and not regress, and this flaw that no longer allows you to download videos in 720p is a setback for the project

We're all volunteers working on this. If you know of a way to stop the "regress[ion]", now is your time to step up to the home plate and help work on it, my friend! Otherwise, your comment is incredibly nonconstructive and demotivating.

**Reply by Alex-Nothoryus** [reply]
@kommunarr Dear colleague, I said that for the purpose I usually use it it is no longer useful, as I only used this project to download videos in 720p, I didn't want to give the impression here that the project is worthless, what's more I said that he is still very good because he blocks advertisements, now if you only want to hear good things I can't do anything, criticism sometimes comes to motivate us in life and not to bring us down, I think you misunderstood me, sorry

**Reply by kommunarr** [reply]
There's a difference between constructive criticism and nonconstructive criticism. That a project "has to progress and not regress" is not substantive feedback that communicates any novel information, especially when the source of the issue is beyond our control.

**Reply by onegladiator4u** [reply]
How Newpipe is still downloading 720p and 1080p with audio ? Are they doing some other way like joining both ?

**Reply by absidue** [reply]
@onegladiator4u Yes, they are doing all the extra work to download an audio and a video stream and to combine them into a single file.
